src/main/java/org/ambraproject/wombat/controller/LockssController.java

```diff
@@ -97,14 +100,29 @@ public String getMonthsForYear(@SiteParam Site site, @PathVariable String year,
     return site + "/ftl/lockss/months";
   }
 
-  @RequestMapping(name = "lockssArticles", value = "/lockss-manifest/vol_{year}/{month}", method = RequestMethod.GET)
+  @RequestMapping(name = "lockssArticles", value = "/lockss-manifest/vol_{year}/{month}",
+      method = RequestMethod.GET, params = {"cursor", "pageNumber"})
   public String getArticlesPerMonth(@SiteParam Site site, @PathVariable String year,
-                                    @PathVariable String month, Model model) throws IOException, ParseException {
-    Map<String, Map> searchResult = (Map<String, Map>) articleArchiveServiceImpl.getArticleDoisPerMonth(site,
-        year, month);
+                                    @PathVariable String month, @RequestParam String cursor,
+                                    @RequestParam String pageNumber, Model model)
+      throws IOException, ParseException {
+    Map<String, ?> searchResult = (Map<String, Map>) articleArchiveServiceImpl
+        .getArticleDoisPerMonth(site, year, month, cursor);
     model.addAttribute("month", month);
     model.addAttribute("year", year);
     model.addAttribute("searchResult", searchResult);
+    model.addAttribute("nextCursorMark", searchResult.get("nextCursorMark"));
+
+    int pageNumberCount = Integer.parseInt(pageNumber);
+    final int listStartNumber = pageNumberCount * MAXIMUM_SOLR_RESULTS + 1;
+    model.addAttribute("listStart", listStartNumber);
+
+    final boolean isLastPage = (Double) searchResult.get("numFound")
+        <= listStartNumber + MAXIMUM_SOLR_RESULTS;
```

Comment:
I think line 121 should change <= to < for comparison.

suppose max-solr-results = 10, and numFound = 61, so results will show in 7 pages. when pageNumber=5 (it is index starting with 0), then listStartNumber = 5*10+1=51 (it is index starting with 1), then isLastPage = (61 <= 51+10) = true, but there is one more page after this one.

Reply:
nice catch! Pushed the change.